packages/babel-plugin-transform-react-jsx/src/index.js

```diff
@@ -1,100 +1,15 @@
+/* eslint-disable-next-line @babel/development/plugin-name */
+import transformStable from "./transform-stable";
+/* eslint-disable-next-line @babel/development/plugin-name */
+import transformExperimental from "./transform-experimental";
 import { declare } from "@babel/helper-plugin-utils";
-import jsx from "@babel/plugin-syntax-jsx";
-import helper from "@babel/helper-builder-react-jsx";
-import { types as t } from "@babel/core";
 
 export default declare((api, options) => {
-  api.assertVersion(7);
+  const runtime = options.runtime || "classic";
```

Comment:
We do throw an error if `runtime` isn't `"classic"` or `"automatic"`. It's in the helper Babel plugin instead of the main plugin though because we'll get rid of this file in v8.0.0. (I inlined all the "classic" code in the new helper Babel plugin)

Reply:
Let's add a comment here to remind ourselves :)